### PR TITLE
Feature Manage WishList

### PR DESCRIPTION
**Pull request by 13XAVI**
This PR is for user to view and manage my wishlist so that he/she can keep track of products he/she is interested in purchasing in the future

### Description of Task to be completed?
 - [ ] Buyers can add products to their wishlist from product pages or search results.
 - [ ] Buyers can view their wishlist at any time to see all the items they have saved.
 - [ ] Buyer can remove item from wish list.
 - [ ] Implement appropriate error handling and responses ,unit tests and documentation
#### http://localhost:3000/api-docs/#/
image
### Tests
image

### CODE REVIEW

**Comment by ambroisegithub** [comment]
Please add your swagger and test screenshot prove and try to test the uncovered lines not tested in your buyer wish-list controller.

**Review by jkarenzi — APPROVED** [review]

File: src/controller/buyerWishlistController.ts
Comment (jkarenzi) [inline src/controller/buyerWishlistController.ts:1]: The code coverage of your controller is low (60%). Try to increase it

**Review by jkarenzi — CHANGES_REQUESTED** [review]

**Review by wayneleon1 — APPROVED** [review]
Looks Good,
but you have to increase your code coverage at least 90%

**Review by wayneleon1 — DISMISSED** [review]

File: src/controller/buyerWishlistController.ts
Comment (TresorXavier) [reply src/controller/buyerWishlistController.ts:1]: code i think codes were lost

**Review by jkarenzi — DISMISSED** [review]

**Review by jkarenzi — APPROVED** [review]

**Review by ambroisegithub — APPROVED** [review]
It Looks good for merge!!!

**Review by ambroisegithub — APPROVED** [review]
Good job!!!

**Pull request merged**